docs/partitioning.txt

```diff
@@ -22,17 +21,15 @@ Explicit graph partitioning can only enabled against storage backends that suppo
 
 There are two aspects to graph partitioning which can be individually controlled: edge cuts and vertex cuts.
 
-Edge Cut
-~~~~~~~~
+=== Edge Cut
```

Comment:
What are you suggesting? Not sure whether you're suggesting that I should add this as a note to the docs, eliminate or discourage edge cuts, so I added it to the issue for following up.